internal/jest-config/transformer.js

```diff
@@ -4,6 +4,7 @@ module.exports = createTransformer({
   presets: [
     ["@babel/preset-env", { targets: { node: "current" } }],
     "@babel/preset-typescript",
+    "@babel/preset-flow",
```

Comment:
how strange! Do you know why this is needed? Is this because react-native needs flow compilation?